bitnami/rabbitmq/templates/secret.yaml

```diff
@@ -0,0 +1,23 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+{{- if or (not .Values.auth.existingErlangSecret) (not .Values.auth.existingSecret) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
+  {{- if (not .Values.auth.existingSecret ) }}
+  rabbitmq-password: {{ include "redis.password" . | b64enc | quote }}
+  {{- end }}
+  {{- if (not .Values.auth.existingErlangSecret ) }}
+  rabbitmq-erlang-cookie: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "rabbitmq-erlang-cookie" "length" 32 "providedValues" (list "auth.erlangCookie") "context" $) }}
+  {{- end }}
+{{- end }}
```

Comment:
Please do not rename this secret template file to reduce the number of changes.

Reply:
do you want me to change it back into one file again also? I changed it to secret to show that I seperated the yamls inside to seperate files. (its now one secrets instead of multiple)

Reply:
this is what I did.